src/diff-snapshot.js

```diff
@@ -69,7 +127,7 @@ function diffImageToSnapshot(options) {
     const totalPixels = imageWidth * imageHeight;
     const diffRatio = diffPixelCount / totalPixels;
 
-    let pass = false;
+    let pass = !hasSizeMismatch;
```

Comment:
This marks test as failed when size is different. Note that at this point diff is always exist, so it's not possible to have 0% diff.

Reply:
what do you mean by `but this is not necessary since resized images are always has diff at this point and would fail the test`?

Reply:
Ah nevermid, just realised it's actually a valid case then size difference is small